### DISCUSSION TITLE
Hide trashed filter for users

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#8726] [Help]
I'm using soft deletes, but don't want my users to be able to see the trashed items of the `resource`. How do I hide the filters depending on the user?

**Comment by leandrocfe** [comment]
You can use `->visible()`

```php
->filters([
    Tables\Filters\TrashedFilter::make()
        ->visible(auth()->user()->is_admin)
])
```

**Reply by bilogic** [reply]
Thanks, does it just hide or actually prevent? I wasn't thinking that clearly when I asked, my goal is to prevent.

**Reply by leandrocfe** [reply]
I think you can use [model policies](https://filamentphp.com/docs/3.x/panels/resources/deleting-records#authorization)
